### PR TITLE
Updates for PureScript 0.11

### PR DESCRIPTION
**Pull request by MonoidMusician**
I wanted to help update this package for PureScript 0.11. I updated the dependencies and changed IsForeign instances to read* functions. I have verified that this builds with the latest compiler.

Thanks!

### CODE REVIEW

**Review by natefaubion — CHANGES_REQUESTED** [review]
Looks good! Just needs an updated `package.json` so Travis can run.

File: src/Ace/Types.purs [hunk 1]
```diff
@@ -25,25 +26,22 @@ readDocumentEventType "remove" = pure Remove
...
-         <*> readProp "lines" e
-    pure $ DocumentEvent r
+readDocumentEvent :: Foreign -> F DocumentEvent
+readDocumentEvent e = do
+  action <- e ! "action" >>= readString >>= readDocumentEventType
+  start <- e ! "start" >>= readPosition
+  end <- e ! "end" >>= readPosition
+  lines <- e ! "lines"
+    >>= readArray
+    >>= (map readString >>> sequence)
```
Comment (natefaubion) [inline src/Ace/Types.purs]: `map f >>> sequence` = `traverse f`

**Review by natefaubion — CHANGES_REQUESTED** [review]

**Comment by natefaubion** [comment]
Looks like the examples/tests need to be updated.

**Comment by MonoidMusician** [comment]
Okay I believe it should be good now. As a bonus I got the Travis build working too ;) Thanks for the tips!

**Review by natefaubion — APPROVED** [review]

**Comment by natefaubion** [comment]
Thanks!

**Pull request merged**